pycoast/cw_base.py

```diff
@@ -32,6 +32,16 @@
 logger = logging.getLogger(__name__)
 
 
+class Proj(pyproj.Proj):
+    """Wrapper around pyproj to add in 'is_latlong'."""
+
+    def is_latlong(self):
+        if hasattr(pyproj.Proj, 'is_latlong'):
```

Comment:
This will probably cause deprecation warnings if 2.0 <= pyproj version < 2.2. In pyresample we check the pyproj version directly in order to avoid warnings.

Reply:
Hm good point. My only argument against that is that now that pyproj 2.2 is available I don't know why you would have 2.0-2.2 and *not* upgrade to 2.2. Maybe it is better to say `if hasattr(self, 'crs')`.

Reply:
I would suggest 
```python
if pyproj.__version__ >= '2':
    return super(Proj, self).is_latlong()
else:
    return self.crs.is_geographic
```

Reply:
If we have a `crs` attribute, why not use it?

Reply:
Oh sorry, didn't see your comment. `hasattr(self, 'crs')` is fine, too